tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: verify-repo-signature
+      image: registry.redhat.io/rhtas/cosign-rhel9@sha256:75791f7ee9257e837d1e3db13415d858c397e803ffa4534c51ca19a2ba34b2ab
+      script: |
+        set -eux
+        MRRC_FILE="$(workspaces.data.path)/$(params.mrrcParamFilePath)"
+        . "$MRRC_FILE"
+        echo "Verify the maven repo zip $MRRC_ZIP_REGISTRY"
+        cosign verify --key /home/konflux/.cosign/cosign.pub $MRRC_ZIP_REGISTRY
+      volumeMounts:
+        - name: "pub-key-vol"
+          mountPath: "/home/konflux/.cosign/"
+    - name: download-maven-repo
+      image: quay.io/konflux-ci/release-service-utils:2a7815cc4bed65918422cf3c9fab4d2d3f298f58
```

Comment:
I know this is a commit based tag, but it still isn't `@sha256`. Can you use one like that? Same for line 47

Reply:
Um...I'm not sure what's different, because I see many other tasks are also using this type of tag. Whatever, I'll update to digest based tag like the cosign one.

Reply:
It is different, because the digest reference really tells you which image it is. A tag can always change in theory (but not really for these sha based tags).

You're right, it's actually the convention of this repo to use the sha tag (and not the `@sha` reference). @johnbieren it should be ok, right?

Reply:
Yes, I thought we used sha tags throughout, but @ligangty is right. Resolving this, sorry for the noise